taskboot/cli.py

```diff
@@ -93,6 +93,9 @@ def main():
         action='append',
         help='Docker build args passed for each built service',
     )
+    compose.add_argument(
+        '--service', action='append', type=str, help='Build only the specific compose service'
```

Comment:
I think it misses the `default=[]` or you'll get the same error as on https://github.com/mozilla/task-boot/pull/27/files

Reply:
We don't get the same error as we have `if args.service` but it's cleaner this way